### PR TITLE
Sphinx documentation pages

### PR DESCRIPTION
**Pull request by dangunter**
Proposed changes:
- [x] Create Sphinx docs project (enable MyST so we can use Markdown) in docs/ directory
- [ ] Update README to point at these docs for more information

### Legal Acknowledgement

By contributing to this software project, I agree to the following terms and conditions for my contribution:

1. I agree my contributions are submitted under the license terms described in the LICENSE.txt file at the top level of this directory.
2. I represent I am authorized to make the contributions and grant the license. If my employer has rights to intellectual property that includes these contributions, I represent that I have received permission to make contributions and grant the required license on behalf of that employer.

### CODE REVIEW

**Comment by ksbeattie** [comment]
@dangunter can you take a look at these failing tests?

File: pyproject.toml [hunk 1]
```diff
@@ -56,7 +53,7 @@ dev = [
     # For adding copyright headers (see addheader.yml and the readme)
     "addheader >= 0.3.0",
     # Get IDAES from main branch in Github
-    #"idaes-pse @ git+https://github.com/IDAES/idaes-pse"
+    "idaes-pse @ git+https://github.com/IDAES/idaes-pse"
```
Comment (lbianchi-lbl) [inline pyproject.toml]: @dangunter @ksbeattie I believe this is the source of the installation errors (and therefore CI failures) and it should be removed - this requirement is specified in `requirements-dev.txt`.

Comment (dangunter) [reply pyproject.toml]: OK, I removed that. But it seems like some of the CI jobs are still failing. I don't know why.

File: docs/home.md [hunk 2]
```diff
@@ -0,0 +1,23 @@
...
+
+## Installation
+
+### User installation
+
+IDAES-UI is installed as part of the IDAES Integrated Platform (IDAES-IP).
+Please see the README of the [IDAES-PSE repository](https://github.com/IDAES/idaes-pse) and/or the full [IDAES install instructions](https://idaes-pse.readthedocs.io/en/stable/tutorials/getting_started/index.html) for details. 
+
+In a nutshell, you want to add *[ui]* to the list of optional dependencies when you run the main IDAES installation command (*pip* or *conda*).
+For example: `pip install idaes[ui]`
```
Comment (lbianchi-lbl) [inline docs/home.md]:
```suggestion
In a nutshell, you want to add `[ui]` to the list of optional dependencies when you run the main IDAES installation command with `pip`.
For example: `pip install "idaes-pse[ui]"`
```

**Review by lbianchi-lbl — APPROVED** [review]

**Pull request merged**